requirements.txt

```diff
@@ -40,5 +40,4 @@ requests
 scikit-image
 seaborn>=0.9.0
 tabulate
-torchvision
-tqdm
```

Comment:
Why did you remove `tqdm` dependency?
It is used by nupic.research frameworks and projects.

Reply:
Must have happened at some point in a merge conflict resolution. requirements.txt shouldn't be touched in this PR. Updating.

Reply:
👍